parser/notes

```diff
@@ -0,0 +1,9 @@
+JJTree:
```

Comment:
Can you format this note to be more reader-friendly? I can hardly parse out how many different topics this is talking about...

Reply:
Done

Reply:
Still didn't quite get what's the purpose of the note. Could they go into the code? What do you want to achieve but putting them in notes here?

Reply:
> Still didn't quite get what's the purpose of the note. Could they go into the code? What do you want to achieve but putting them in notes here?

These are explaining some things that are specific to Javacc and why we did it that way. Part of it is for giving pointers to future maintainers on how to do things.

Reply:
So for people who understands javacc they will know what you mean?